release/train_tests/benchmark/config.py

```diff
@@ -39,6 +39,9 @@ class BenchmarkConfig(BaseModel):
     max_failures: int = 0
 
     task: str = "image_classification"
+    locality_with_output: bool = True
```

Comment:
locality_with_output doesn't seem useful for this benchmark. we can remove it, or make it default to False